README.md

```diff
@@ -136,7 +136,7 @@ ws.on('error', err => {
   console.error('ERR', err);
 });
 ```
-See inverse [websocket-client.js](./lib/websocket-client.js) & [ws api docs](./doc/websocket-client.md) for further information.
+See inverse [websocket-client.ts](../typescript/src/websocket-client.ts) & [ws api docs](../typescript/doc/websocket-client.md) for further information.
```

Comment:
Same here, should just point to `./src` without the branch name. That way it stays relative to the currently viewed branch.